core/src/main/java/io/questdb/cairo/wal/WalEventCursor.java

```diff
@@ -373,16 +373,28 @@ protected void read() {
     public class InvalidationInfo {
```

Comment:
nit: `MatViewInvalidationInfo` might be a more obvious name. Same with `DataInfoExt` - it should either contain generic `long0`, `long1` fields or be called `MatViewDataInfo`

Reply:
renamed: https://github.com/questdb/questdb/pull/5519/commits/3b0722404833a5b508a78aebc477d80ddd3ccc5e